src/Vizkit3dWorld.cpp

```diff
@@ -270,11 +287,9 @@ void Vizkit3dWorld::setCameraPose(base::samples::RigidBodyState pose) {
      * - Y left
      * - Z up
      */
-    osg::Matrixd m = poseMatrix *
-                     osg::Matrixd::rotate(M_PI_2, osg::Vec3(0.0, 0.0, 1.0)) *
-                     osg::Matrixd::rotate(-M_PI_2, osg::Vec3(1.0, 0.0, 0.0));
-
-
+    osg::Matrixd m = poseMatrix* 
```

Comment:
You need to at least put it in a different commit that explains the change. Given how small the changes are, a different PR is not needed.